src/Mobile/eShopOnContainers/eShopOnContainers.Core/GlobalSettings.cs

```diff
@@ -4,7 +4,7 @@ public class GlobalSetting
     {
         public const string AzureTag = "Azure";
         public const string MockTag = "Mock";
-        public const string DefaultEndpoint = "http://13.88.8.119";
+        public const string DefaultEndpoint = "http://52.170.199.82";
```

Comment:
This is a temporal endpoint. Change this for something like "INSERT YOUR ENDPOINT".

Reply:
Done!